Cartfile

```diff
@@ -1,2 +1,2 @@
-github "BoltsFramework/Bolts-ObjC" ~> 1.9.1
-github "facebook/facebook-ios-sdk" == 11.0.1
+github "vazarkevych/Bolts-ObjC" "spm"
```

Comment:
What is the difference between the https://github.com/BoltsFramework/Bolts-ObjC and your fork?
We should host a fork in the Parse Platform org and link to that.

Reply:
Thank you for your comment, I used Bolts from the original repository